### ISSUE TITLE
Solver is not working in example file

### ISSUE THREAD

**Issue by juliusang** [petrasvestartas/compas_wood#15]
Solver throws the error 1. Error begin run: 'NoneType' object has no attribute 'write' [16:1]

![image](https://github.com/petrasvestartas/compas_wood/assets/63193796/59c3a9d4-b84a-467d-8540-719d74f10bd6)

**Comment by petrasvestartas** [comment]
Can you provide further details shat you did?
Did you follow the installation video from food4rhino and first added install component?
Which Rhino version?

**Comment by juliusayz999** [comment]
Yes I followed the installation video from food4rhino. The example files 1-6 worked without issues. But I faced this error with the solver component when I opened up example file 7.

I am currently using version 8.8
